robocop/checkers/comments.py

```diff
@@ -150,19 +151,14 @@ def check_invalid_comments(self, name, node):
 
     def check_comment_content(self, token, content):
         content = content.lower()
-        if "todo" in content:
+        todos = self.param("todo-in-comment", "todos").split(",")
```

Comment:
Probably there is a better place to construct `todos` list than here for every comment.

Reply:
This is gonna get complicated soon, but in such case you can create custom converter. Like you noticed you cannot construct todos list in checker ``__init__`` because init will be called before rule is configured (so we will not get updated todos list). 

I was in similar situation with ``inconsistent-assignment`` rule - I wanted user to configure it using user friendly names (like 'equal_sign') but for checker purposes I wanted to save different value internally. For that reason I created custom converter:
https://github.com/MarketSquare/robotframework-robocop/blob/111b1f7610e4fd7dfef718840a0639db2dbdd71e/robocop/checkers/misc.py#L138
Anytime the rule is configred (also with default value) the converter is called and converter return value is saved internally instead.

You could do:

def csv_string_to_set(csv_string):
    return set(csv_string.split(",") 
    
 and use it as converter

Reply:
I guess we could stand with following solution:
```python
        if self.todos:
            pass  # avoid resolving todos for each and every comment separately.
        else:
            self.todos = self.param("todo-in-comment", "todos").split(",")
```

Reply:
What about property then? or method:
```
def __init__(self):
    self._todos = None

def todos(self):
    if not self._todos:
        self._todos = self.param("todo-in-comment", "todos").split(",")
    return self._todos
```